.circle/configure-test-rabbitmq.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+sudo sh -c 'echo "[{rabbit, [{disk_free_limit, 10}, {loopback_users, []}, {tcp_listeners, [{\"0.0.0.0\", 5672}]}]}]." > /etc/rabbitmq/rabbitmq.config'
```

Comment:
Yup. Will try to rewrite with that approach. Guess I was being lazy :)

If you are suggesting I use that file then it will not work. Need to explicitly specific the interface to listen on else it ends up on ipv6 and then messes with test config.

If you are saying this already happens elsewhere and I need to fix that then I am happy to change existing scripts too.

So, of the three above which is your suggestion? Perhaps its somethign else entirely :)